src/west/commands/project.py

```diff
@@ -379,13 +408,20 @@ def _projects(args, listed_must_be_cloned=True, include_west=False):
     # include_west (default: False):
     #   If True, west may be given in args.projects without raising errors.
     #   It will be included in the return value if args.projects is empty.
+    #
+    # exclude_manifest (default: False):
+    #   If True, the manifest project will not be included in the returned
+    #   list.
 
     projects = _all_projects()
     west_project = _west_project()
 
     if include_west:
         projects.append(west_project)
 
+    if exclude_manifest:
+        projects.pop(0)
```

Comment:
By the discussion in the earlier series, we technically should be using MANIFEST_PROJECT_INDEX.

Things like this are why I think it's worthwhile to just document that it's at index number zero. But I'm not asking for a change here.

Reply:
You're absolutely right.
Personally I dislike magic numbers myself, so will change to MANIFEST_PROJECT_INDEX.

Reply:
Zero is a sufficiently magic number that I think it's okay to use sometimes.
But I agree being consistent is better.

Reply:
Agree with @mbolivar